dags.yaml

```diff
@@ -1280,3 +1280,23 @@ bqetl_review_checker:
   tags:
     - impact/tier_1
     - repo/bigquery-etl
+
+bqetl_mobile_feature_usage:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - rzhao@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
```

Comment:
In general I'd suggest we keep this to false, unless we expect someone to respond immediately when retrying.